Source/Classes/PINURLSessionManager.m

```diff
@@ -48,6 +48,13 @@ - (void)invalidateSessionAndCancelTasks
     [self unlock];
 }
 
+- (nonnull NSURLSessionDataTask *)dataTaskWithRequest:(nonnull NSURLRequest *)request
+                                    completionHandler:(nonnull PINURLSessionDataTaskCompletion)completionHandler 
+{
+    return [self dataTaskWithRequest:request priority:PINRemoteImageManagerPriorityDefault
```

Comment:
Let's put `priority:PINRemoteImageManagerPriorityDefault` in a new line.

Reply:
It is done :)